### PR TITLE
add method to set & get device orientation

### PR DESCRIPTION
**Pull request by vrunoa**
add isScreenLandscape, isScreenPortrait methods

add setOrientation methods

add unit tests for implemented methods: isScreenLandscape, isScreenPortrait, setOrientation, beLandscape, bePortrait

add setAutoRotationState method

add setAutoRotationState test

add functional test for screen orientation change

update readme with new methods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,3 +148,9 @@ console.log(await adb.getPIDsByName('m.android.phone'));
 - uninstallApk
 - installFromDevicePath
 - install
+- setAutoRotationState
+- isScreenLandscape
+- isScreenPortrait
+- setOrientation
+- beLandscape
```
Comment (imurchie) [inline README.md]: I'm not a big fan of `beXXX` method names. There is nothing else in the codebase like that. I'd prefer just the `setOrientation` method, and the two orientations having constant values. Something like

```js
const orientations = {
  LANDSCAPE: 1,
  PORTRAIT: 0
}
```

**Comment by imurchie** [comment]
There is a problem with the function test

```
1) adb commands should change screen orientation:
     AssertionError: expected false to be true
      at doAsserterAsyncAndAddThen (node_modules/chai-as-promised/lib/chai-as-promised.js:296:33)
      at [object Object].<anonymous> (node_modules/chai-as-promised/lib/chai-as-promised.js:286:25)
      at [object Object].Object.defineProperty.get (node_modules/chai/lib/chai/utils/overwriteProperty.js:50:37)
      at Suite.callee$1$0$ (test/functional/adb-commands-e2e-specs.js:106:26)
      at tryCatch (node_modules/babel-runtime/regenerator/runtime.js:67:40)
      at GeneratorFunctionPrototype.invoke [as _invoke] (node_modules/babel-runtime/regenerator/runtime.js:315:22)
      at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (node_modules/babel-runtime/regenerator/runtime.js:100:21)
      at GeneratorFunctionPrototype.invoke (node_modules/babel-runtime/regenerator/runtime.js:136:37)
      at bound (domain.js:287:14)
      at GeneratorFunctionPrototype.runBound (domain.js:300:12)
      at run (node_modules/core-js/library/modules/es6.promise.js:104:47)
      at node_modules/core-js/library/modules/es6.promise.js:115:28
      at flush (node_modules/core-js/library/modules/$.microtask.js:19:5)
```

**Comment by imurchie** [comment]
What is the purpose of these methods? Do they have benefits over and above the UiAutomator methods that are currently being used to handle the orientation?

**Comment by vrunoa** [comment]
sorry @imurchie I didn't see those methods. My bad. I'll close this PR and use them. Where are those methods defined, under which library ?

**Comment by imurchie** [comment]
They are in `appium-android-driver` (https://github.com/appium/appium-android-driver/blob/master/lib/commands/actions.js#L27-L35) and `appium-android-bootstrap` (https://github.com/appium/appium-android-bootstrap/blob/master/bootstrap/src/io/appium/android/bootstrap/handler/Orientation.java)

**Comment by vrunoa** [comment]
Cool. Just check them out. I don't think my implementation won't have any advantage over the one in ```appium-android-bootstrap``` rather than having the possibility to get(or set) the params directly from ```appium-adb``` in case someone is planning to use ```appium-adb``` module alone.

**Pull request closed without merging**